### PR TITLE
Update Jenkinsfile

### PR DESCRIPTION
**Pull request by zFernand0**
Signed-off-by: zFernand0 <fernando.rijocedeno@broadcom.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -116,17 +108,21 @@ node('ca-jenkins-agent') {
...
-                sh "cp __tests__/__resources__/properties/example_properties.yaml __tests__/__resources__/properties/custom_properties.yaml"
-                sh "npm run test:integration"
-            },
-            testResults: [dir: "${INTEGRATION_TEST_ROOT}/jest-stare", files: "index.html", name: "${PRODUCT_NAME} - Integration Test Report"],
-            junitOutput: INTEGRATION_JUNIT_OUTPUT,
-        )
+    pipeline.test(
+        name: "Integration",
+        operation: {
+            sh "npm i -g @zowe/cli@daily --zowe:registry=${pipeline.registryConfig[0].url}"
```
Comment (tucker01) [inline Jenkinsfile]: Is this supposed to be daily?

Comment (zFernand0) [reply Jenkinsfile]: I made it `latest` since we will be getting rid of daily hopefully pretty soon (by next week)

Comment (zFernand0) [reply Jenkinsfile]: Forget my previous comment.
It should be `latest`
Thanks for catching that. I thought we were talking about the dependencies.

**Comment by zFernand0** [comment]
@tucker01, feel free to check this out again!
Thank you for catching that missing daily reference

**Pull request merged**